misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -4785,6 +4787,11 @@ private NodeList<ImportDeclarationNode> sortAndGroupImportDeclarationNodes(
             NodeList<ImportDeclarationNode> importDeclarationNodes) {
         // moduleImports would collect only module level imports if grouping is enabled,
         // and would collect all imports otherwise
+        if (importDeclarationNodes.size() == 0) {
+            return importDeclarationNodes;
+        }
```

Comment:
Shall we also have this before the comment above? Since the comment is not related to this early exit. 
```suggestion
        if (importDeclarationNodes.isEmpty() == 0) {
            return importDeclarationNodes;
        }

```